extensions/objectify/src/main/java/com/github/nmorel/gwtjackson/objectify/client/KeyJsonDeserializer.java

```diff
@@ -0,0 +1,35 @@
+package com.github.nmorel.gwtjackson.objectify.client;
+
+import com.github.nmorel.gwtjackson.client.JsonDeserializationContext;
+import com.github.nmorel.gwtjackson.client.JsonDeserializer;
+import com.github.nmorel.gwtjackson.client.JsonDeserializerParameters;
+import com.github.nmorel.gwtjackson.client.stream.JsonReader;
+import com.github.nmorel.gwtjackson.objectify.shared.KeyConstant;
+import com.googlecode.objectify.Key;
+
+public class KeyJsonDeserializer<T> extends JsonDeserializer<Key<T>> {
+
+    private static final KeyJsonDeserializer INSTANCE = new KeyJsonDeserializer();
+
+    public static <T> KeyJsonDeserializer<T> newInstance( JsonDeserializer<T> deserializer ) {
+        return INSTANCE;
+    }
+
+    public static <T> KeyJsonDeserializer<T> getInstance() {
```

Comment:
this one is used ?

Reply:
Yes it's used in RefJsonDeserializer#L36